frontends/precinct-scanner/src/screens/scan_error_screen.tsx

```diff
@@ -17,29 +17,33 @@ export function ScanErrorScreen({ error, isTestMode }: Props): JSX.Element {
   const errorMessage = (() => {
     if (!error) return undefined;
     switch (error) {
-      // Invalid interpretation
+      // Invalid ballot interpretations
       case 'invalid_test_mode':
-        return isTestMode ? 'Live Ballot detected.' : 'Test ballot detected.';
+        return isTestMode
+          ? 'Live ballot detected. Scanner is in test mode.'
+          : 'Test ballot detected. Scanner is in live mode.';
       case 'invalid_election_hash':
-        return 'Scanned ballot does not match the election this scanner is configured for.';
+        return 'The ballot does not match the election this scanner is configured for.';
       case 'invalid_precinct':
-        return 'Scanned ballot does not match the precinct this scanner is configured for.';
+        return 'The ballot does not match the precinct this scanner is configured for.';
       case 'unreadable':
-        return 'There was a problem reading this ballot. Please try again.';
       case 'unknown':
-        return undefined;
-      // Precinct scanner error
+        return 'There was a problem reading this ballot. Please scan again.';
+      // Precinct scanner errors
       case 'scanning_failed':
+        return 'Scanning Failed. Remove ballot to continue.';
```

Comment:
This case happens when the scanner repeatedly tries to pull the ballot in but it doesn't work (e.g. if you hold onto the ballot and don't let it scan). Eventually we give up trying to scan it.

I think we can recommend removing the ballot and trying again.

Reply:
How about this?
> Ballot not fully inserted. Remove ballot to continue.

FWIW, I have removed all the "and try again" language because I feel that inserting a ballot and trying again is implicit the use of a ballot scanner. Additionally, once the voter follows the direction "Remove ballot to continue" the screen updates to be the "Insert Ballot" screen.

I'm going to commit this assuming you'll agree. However I expect you'll tell me if otherwise! 😄

Reply:
Sounds good!